src/main/java/net/dv8tion/jda/api/interactions/components/selections/EntitySelectMenu.java

```diff
@@ -108,6 +114,16 @@ default EntitySelectMenu withDisabled(boolean disabled)
     @Nonnull
     EnumSet<ChannelType> getChannelTypes();
 
+    /**
+     * Default selected values.
+     * <br>These are shown until the user customizes the selected values,
+     * which then fires a {@link SelectMenuInteraction}.
```

Comment:
```suggestion
     * which then fires a {@link SelectMenuInteractionEvent}.
```
Might need the import